### ISSUE TITLE
M1 Mac Install

### ISSUE THREAD

**Issue by jasheldo** [coiled/talkpython-getting-started-with-dask#2]
I have an M1 Mac and installing the packages is problematic. I can't use miniconda or Anaconda. Homebrew and pip are what I have. I've been able to install most of the packages by building them from source but am completely unable to get scikit-learn to install. Is there any other solution around this?

**Comment by mikeckennedy** [comment]
Hi @jasheldo 

Not sure what the trouble is, but I got it working fine on my M1. Here are some steps:

<img width="770" alt="Screen Shot 2021-07-12 at 4 53 20 PM" src="https://user-images.githubusercontent.com/2035561/125369724-b684ef00-e331-11eb-961e-7dd387c30584.png">

After setting up Jupyter to use the venv and installing dask + distributed there, I was able to choose that env in Jupyter and run with it.

**Comment by jasheldo** [comment]
Thank you for your time with this @mikeckennedy. I've no issues with the steps listed above. I looked into the `environment.yml` file and started installing components from there. I've been unable to `pip install` some packages (i.e. `scipy`) but can build them from source and install that way. No issue. But `scikit-learn` I've been wholly unable to build or install. Let me start going through the course and if it remains problematic I'll reach back out.

**Comment by mikeckennedy** [comment]
Hi @jasheldo You're welcome. Glad those steps work. I was able to install `scikit-learn` on mine just using pip:
<img width="812" alt="Screen Shot 2021-07-13 at 8 56 56 AM" src="https://user-images.githubusercontent.com/2035561/125485209-1fceb30a-74bf-4df0-8373-5b771757109d.png">

Double check what version of Python you're using. I am using Python 3.9.6 but I am running the Intel rather than M1 install of Python so maybe that's the difference?

Basically I installed Homebrew using a Rosetta Terminal, that got me Intel Homebrew. Then Homebrewed Python which gave me Intel Python. I did this because there wasn't an M1 Homebrew when I setup my machine.
